client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -409,7 +409,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 
 	function pickDesign( _selectedDesign: Design | undefined = selectedDesign ) {
 		setSelectedDesign( _selectedDesign );
-		if ( siteSlugOrId && _selectedDesign ) {
+		if ( ( isFreeFlow( flow ) || siteSlugOrId ) && _selectedDesign ) {
```

Comment:
Instead of using isFreeFlow(), we can pass in a prop like, useWithoutSite, to make this component work without any siteSlug/siteId.

Reply:
Yes I agree. Steps shouldn't be aware of the concept of "flow" when possible. This makes them reusable.